src/Diagnostics.DataProviders/DataProviderConstants.cs

```diff
@@ -15,6 +15,8 @@ public class DataProviderConstants
 
         public const string AzureUSGovernment = "Government";
 
+        public const string KeyVaultURL = "https://appservicediagnostics.vault.azure.net/";
```

Comment:
1. Is this the right file for this Const.
2. Ideally, this should be coming from AppSettings (overridden for different environments) ?

Reply:
I was planning to separate prod from dev as this is only Dev. and the key vault is loaded at the same time as the configuration files, there is a workaround but it would result in a slow down of startup times.

Reply:
would you still like me to make the constant come from AppSettings?

Reply:
I believe you would have to do that eventually since this endpoint would need to be configurable for dev/prod/mooncake